webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardList.vue

```diff
@@ -211,6 +214,44 @@ export default {
     loadNextPage() {
       this.originalLimitToFetch = this.limitToFetch += this.pageSize;
     },
+    getUsersByadvancedfilter(profileSettings) {
```

Comment:
can we combine this function with searchPeople function since it has quite the same code ?

Reply:
is not the some code , i think we can't combine this function , we have different condition to manage the simple search result and  the advanced search result !